src/Traits/VerifiableTrait.php

```diff
@@ -15,20 +14,33 @@ trait VerifiableTrait
 
     protected function loadActiveVerifications(string $action)
     {
-        $newVerifications = $this->shouldKeepDuringSession($action)
-            ? VerificationCode::allFor($this)->whereNull('verifies_until')->whereNotNull('used_at')->get()
-            : VerificationCode::allFor($this)->where('verifies_until', '>', Carbon::now()->toDateTime())->get();
+        $newVerifications = $this->getNewVerifications($action);
 
         $this->activeVerifications = is_null($this->activeVerifications)
             ? $newVerifications
             : $this->activeVerifications->merge($newVerifications);
     }
 
-    private function shouldKeepDuringSession(string $action): bool
+    private function getNewVerifications(string $action): Collection
     {
-        return (Config::get('verifications.actions.'.$action.'.keep_verified_during_session') === true)
-            && Session::has('last_activity')
-            && (Session::get('last_activity') < Carbon::now()->addMinutes(Config::get('session.lifetime'))->toDateTime());
+        $now = Carbon::now();
+
+        switch (Config::get('verifications.actions.'. $action .'.expires_from')) {
+            case 'last-activity':
```

Comment:
Možno by som to dala do enumu alebo do nejakej konštanty keď sa to opakuje na viac miestach

Reply:
Toto sme s Kikou riesili na slacku, ze by to bolo fajn, ale potrebovali by sme nainstalovat do package spatie/enum. Cize vec nazoru, ak chceme rozsirovat package dependency, bude to asi krajsie 💁

Reply:
Ne netreba, nechajme takto